src/ralph/lib/permissions/models.py

```diff
@@ -12,31 +12,133 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+def get_perm_key(action, class_name, field_name):
+    """
+    Generated django permission code name.
+
+    :Example:
+
+        >> perm = get_perm_key('change', 'baseobject', 'remarks')
+        >> print(perm)
+        >> change_baseobject_remarks_field
+
+    :param action: Permission action (change/view)
+    :param class_name: Django model class name
+    :param field_name: Django model field name
+
+    :type action: str
+    :type class_name: str
+    :type field_name: str
+
+    :return: django permission code name
+    :rtype: str
+    """
+    return '{}_{}_{}_field'.format(action, class_name, field_name)
+
+
 class PermissionByFieldBase(ModelBase):
-    """Extends permissions by edit per field of model."""
+
+    """
+    Metaclass added django permissions based on all fields in the model.
+
+    :Example:
+
+        class Test(with_metaclass(PermissionByFieldBase, models.Model)):
+
+            ...
+
+            class Permissions:
+                # Fields to exclude generated permissions
+                blacklist = set(['service_env'])
```

Comment:
don't incorporate servie_env here - rename it to `sample_field` or sth like that